packages/edit-site/src/components/page-pages/index.js

```diff
@@ -147,17 +139,12 @@ export default function PagePages() {
 				id: 'actions',
 				cell: ( props ) => {
 					const page = props.row.original;
-					return (
-						<PageActions
-							postId={ page.id }
-							onRemove={ () => setResetQuery() }
-						/>
-					);
+					return <PageActions postId={ page.id } />;
```

Comment:
In my tests the onRemove is not necessary core-data automatically updates the queries when a record is removed.

Reply:
For now it can be removed, yes. That was before adding `onPaginationChange`.